uat/custom-components/client-java-paho/src/main/java/com/aws/greengrass/testing/util/TimeUtil.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.testing.util;
+
+public final class TimeUtil {
```

Comment:
shouldn't need this class, java provides TimeUnit.SECONDS.toMillis(seconds), etc

Reply:
fixed